.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,49 @@
+# Code of Conduct
+
+"We promise to extend courtesy and respect to everyone involved in this project regardless of gender, gender identity, sexual orientation, disability, age, ethnicity, religion, level of experience, or any other aspect of life. We expect anyone contributing to the Yeoman project to do the same."
+
+All contributions to this project are valuable, including and not limited to posting issues, updating documentation, submitting pull requests and providing feedback in comments. Please act with respect and avoid demeaning, condescending, racist, sexist and other inappropriate language and conduct. 
+
+Please ensure comments stay professional, constructive and about the code, not the person. 
+
+[![xkcd](xkcd-1053.png)](http://xkcd.com/1053/)
```

Comment:
❤️ love this comment, +1!

Touching up the alt text to make it clear it's this comic:

```suggestion
[![xkcd comic 1053](xkcd-1053.png)](http://xkcd.com/1053/)
```